core/src/test/kotlin/javadoc/JavaAsciidocTest.kt

```diff
@@ -0,0 +1,114 @@
+package javadoc
+
+import com.google.inject.Guice
+import com.intellij.util.io.delete
+import org.jetbrains.dokka.*
+import org.jetbrains.dokka.Utilities.DokkaOutputModule
+import org.jetbrains.dokka.tests.appendDocumentation
+import org.junit.Test
+import java.io.File
+import java.nio.file.Files
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+/**
+ * @author Mario Toffia
+ */
+class JavaAsciidocTest {
+    @Test fun javaAsciidocJavaSourceGetsRendered() {
+        val str = generate("testdata/format/asciidocJavaSource.kt", "AsciidocJavaSourceKt.html")
+
+        assertTrue {
+            str.contains("<pre class=\"CodeRay highlight\"><code data-lang=\"java\"><span " +
```

Comment:
@semoro Is it possible for I can get the inline HTML in this pull request. I will have one more in august where I'll add more unit test and I put the HTML in separate files. Is this OK? If no I'll fix it :)